### PR TITLE
update docs link

### PR DESCRIPTION
**Pull request by elbeejay**
update link to documentation in pyproject.toml file

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ doc = [
 
 [project.urls]
 homepage = "https://github.com/USGS-python/dataretrieval"
```
Comment (thodson-usgs) [inline pyproject.toml]:
```suggestion
homepage = "https://github.com/DOI-USGS/dataretrieval-python"
```

File: pyproject.toml [hunk 2]
```diff
@@ -36,7 +36,7 @@ doc = [
...
-documentation = "https://usgs-python.github.io/dataretrieval"
+documentation = "https://doi-usgs.github.io/dataretrieval-python/"
 repository = "https://github.com/USGS-python/dataretrieval.git"
```
Comment (thodson-usgs) [inline pyproject.toml]:
```suggestion
repository = "https://github.com/DOI-USGS/dataretrieval-python.git"
```

**Review by thodson-usgs — CHANGES_REQUESTED** [review]
See suggestions.

File: pyproject.toml (hunk 2 above)
Comment (elbeejay) [reply pyproject.toml]: wow duh - thank you for catching these!

**Review by thodson-usgs — APPROVED** [review]

**Pull request merged**